packages/webpack-cli/lib/bootstrap.js

```diff
@@ -49,14 +49,21 @@ async function runCLI(cli, commandIsUsed) {
             // if the unknown arg starts with a '-', it will be considered
             // an unknown flag rather than an entry
             let entry;
-            if (parsedArgs.unknownArgs.length === 1 && !parsedArgs.unknownArgs[0].startsWith('-')) {
-                entry = parsedArgs.unknownArgs[0];
-            } else if (parsedArgs.unknownArgs.length > 0) {
-                parsedArgs.unknownArgs
-                    .filter((e) => e)
-                    .forEach((unknown) => {
-                        logger.warn('Unknown argument:', unknown);
+            if (parsedArgs.unknownArgs.length > 0 && !parsedArgs.unknownArgs[0].startsWith('-')) {
+                if (parsedArgs.unknownArgs.length === 1) {
+                    entry = parsedArgs.unknownArgs[0];
+                } else {
+                    entry = [];
+                    parsedArgs.unknownArgs.forEach((unknown) => {
```

Comment:
`webpack --entry a.js b.js c.js`  is what we should be supporting imo.

Reply:
It will be a bit tricky to parse `--entry` like  that.

Reply:
We need to discuss which to support or support both.
/cc @webpack/cli-team

Reply:
@rishabh3112 I think we support two cases here for better 0CJS:
```
webpack --entry ./src/index.js --entry ./src/other.js
webpack ./src/index.js ./src/other.js
```

Second just sugar.

Reply:
I agree with @evilebottnawi we should support both.

Reply:
Lets do both then. But this would be a breaking change so should be mentioned as such during release.

Reply:
@rishabh3112 Why? It is breaking change?

Reply:
Yeah it's not, my bad. I wrongly thought that we were supporting configurations to be passed without flags instead on entry in CLI v3.

Reply:
Configuration should loaded by default (0CJS) or using `--config` flag